src/main/java/com/TableTalk/Enterprise/dao/GameDAO.java

```diff
@@ -13,19 +13,20 @@
 
 
 @Repository
-public class GameDAO implements IGameDAO{
+public class GameDAO implements IGameDAO {
 
     private static String CLIENT_ID = "ASOEAibUZS";
```

Comment:
This needs to be commented, as someone who has never seen your code, it's totally foreign.